### ISSUE TITLE
No build possible with GHC 7.10 because of base-constraint.

### ISSUE THREAD

**Issue by NobbZ** [anchor/git-vogue#67]
It is a pity, that this problem seems not only related to git-vogue, but your dependencies as well.

**Comment by christian-marie** [comment]
Might be able to drop the troublesome ones. The base constraint should be able to be loosened. I'll get travis building against 7.10 as I consider this a problem.

**Comment by christian-marie** [comment]
Work has begun in #68, currently waiting on an email back from the list-tries author. Alternately I can probably write a simple algorithm to do the same thing as the somewhat convoluted code currently there.

**Comment by christian-marie** [comment]
The list-tries author is waiting on https://ghc.haskell.org/trac/ghc/ticket/9971 to be merged into the 7.10 rc.

**Comment by christian-marie** [comment]
Waiting on this now:

https://github.com/jaspervdj/stylish-haskell/issues/77

**Comment by NobbZ** [comment]
That one is fixed, but needs to get released on hackage.
Am 21.01.2015 08:54 schrieb "Christian Marie" <notifications@github.com>:

**Comment by christian-marie** [comment]
Now just waiting on ghc-mod to build on 7.10.

**Comment by NobbZ** [comment]
It does! Or at least it did for me... I've installed ghc-mod on my system. Can't verify right now, since I'm not at home.

**Comment by expipiplus1** [comment]
This can be closed now, everything builds without issue on 7.10.3.

**Comment by ctd** [comment]
@expipiplus1 thanks for the update.
I'll close this issue.
